dracut-saltboot/saltboot/saltboot.sh

```diff
@@ -294,6 +296,15 @@ while kill -0 "$SALT_PID" >/dev/null 2>&1; do
     kill "$SALT_PID"
     sleep 1
   fi
+  #detect salt kill message
+  if [ -f "$SALT_STOP" ];then
+    snum=$(( snum + 1 ))
+    if [ "$snum" -gt 15 ];then
```

Comment:
Please make this configurable live we have with `SALT_TIMEOUT`. Something like `SALT_STOP_TIMEOUT` kernel option.